.github/workflows/copyrights.yml

```diff
@@ -0,0 +1,22 @@
+name: copyrights
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  copyrights:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: yegor256/copyrights-action@0.0.8
+        with:
+          license: LICENSE.txt
+          globs: >-
+            **/*.yml
+            **/*.yaml
+            **/*.hs
+            **/*.sh
+            **/*.phi
```

Comment:
Exclude `.phi` files. It's impossible to add copyright notice there since their format doesn't support comments - see https://github.com/objectionary/eo-phi-normalizer/issues/450#issuecomment-2478540817.